src/main.lua

```diff
@@ -1101,6 +1102,68 @@ addEventingHandler(
 	end
 )
 
+addEventingHandler(
```

Comment:
ah, I see it now - i don't think we need a separate handler for instant - so long as we accept the tag on existing decrease-stake handlers (operators and delegates) and the logic matches

Reply:
So should we add Vault-ID in here too?  I could still combine the logic in some utility functions but maintain separate handlers.